internal/reports/report_helpers.go

```diff
@@ -546,10 +556,11 @@ var IssueDB = map[internal.SchemaIssue]struct {
 		CategoryDescription: "Primary Key is missing, synthetic column created as a primary key"},
 	internal.UniqueIndexPrimaryKey: {Category: "UNIQUE_INDEX_PRIMARY_KEY",
 		CategoryDescription: "Primary Key is missing, unique column(s) used as primary key"},
-	internal.ArrayTypeNotSupported: {Brief: "Array datatype migration is not fully supported. Please validate data after data migration", Severity: warning, Category: "ARRAY_TYPE_NOT_SUPPORTED"},
-	internal.SequenceCreated:       {Brief: "Auto Increment has been converted to Sequence, set Skipped Range or Start with Counter to avoid duplicate value errors", Severity: warning, Category: "SEQUENCE_CREATED"},
-	internal.ForeignKeyOnDelete:    {Brief: "Spanner supports only ON DELETE CASCADE/NO ACTION", Severity: warning, Category: "FOREIGN_KEY_ACTIONS"},
-	internal.ForeignKeyOnUpdate:    {Brief: "Spanner supports only ON UPDATE NO ACTION", Severity: warning, Category: "FOREIGN_KEY_ACTIONS"},
+	internal.ArrayTypeNotSupported:         {Brief: "Array datatype migration is not fully supported. Please validate data after data migration", Severity: warning, Category: "ARRAY_TYPE_NOT_SUPPORTED"},
+	internal.SequenceCreated:               {Brief: "Auto Increment has been converted to Sequence, set Skipped Range or Start with Counter to avoid duplicate value errors", Severity: warning, Category: "SEQUENCE_CREATED"},
+	internal.ForeignKeyOnDelete:            {Brief: "Spanner supports only ON DELETE CASCADE/NO ACTION", Severity: warning, Category: "FOREIGN_KEY_ACTIONS"},
+	internal.ForeignKeyOnUpdate:            {Brief: "Spanner supports only ON UPDATE NO ACTION", Severity: warning, Category: "FOREIGN_KEY_ACTIONS"},
+	internal.ForeignKeyActionsNotSupported: {Brief: "Spanner doesn't support foreign key action migration", Severity: warning, Category: "FOREIGN_KEY_ACTIONS"},
```

Comment:
The message is misleading since Spanner does not support any migration, it should be FK Action migration not supported for source oracle/ onyl support for mysql/pg by SMT, something along these lines.